transaction-status/src/parse_token/extension/confidential_mint_burn.rs

```diff
@@ -0,0 +1,256 @@
+use {
```

Comment:
why are we introducing so much new code in a "version bump" pr backport?

Reply:
The new token program is deployed already (or will be deployed), but the parsers in RPC need to be updated to handle those, or we get complaints. In the case of the token-group extension, since there was a change to the data types, RPC currently returns total nonsense.

You can see https://github.com/solana-labs/solana/pull/34412 for a previous example where we bring SPL and the monorepo in line. All of the new code is exactly the same kind.

Once we break out RPC from the monorepo or even just these crates, we won't have to do this though.